docs: Improve the API reference of all the components

selemondev opened the pull request:
This pull request enhances Shadcn-Vue's documentation by providing comprehensive documentation for all its components, closely aligned with Shadcn UI.

Closes #145

selemondev committed:
docs: improve the api reference of all the components

zernonia commented on apps/www/src/lib/registry/default/example/AlertDestructiveDemo.vue:
let's move this component to `new-york` as it is using `new-york` style. Or perhaps create another `default` one?

selemondev replied:
Done.

zernonia commented on apps/www/src/lib/registry/default/example/BadgeDestructiveDemo.vue:
can we also duplicate these default styling demo for `new-york` style? Else the component preview and codeeditor would fail.

selemondev replied:
Done 🚀 .

zernonia commented:
Awesome job @selemondev ! Just some file location changes, and some missing component for `new-york` style. 😁

selemondev commented:
> Awesome job @selemondev ! Just some file location changes, and some missing component for `new-york` style. 😁

All the component demos now support both default and new-york theme 😄.

selemondev committed:
docs(app): #152 add new-york theme

This pull request is intended to add the new-york theme for the newly created component examples.

Closes: #152

zernonia committed:
chore: build registry

zernonia committed:
chore: change usage of lucide icon in new-york

zernonia reviewed (approved):
LGTM! Great job @selemondev !
